.github/ISSUE_TEMPLATE/FEATURE.md

```diff
@@ -0,0 +1,19 @@
+---
+name: 🌈 Feature
+about: What cool thing would you like to add?
+title: 'Feature request: '
+labels: 'feature request'
+---
+
+# 🌈 Feature
+
+## Packages affected
+
+- [ ] Sandpack-client
+- [ ] Sandpack-react
+
+## What is this feature?
+
+## How the feature should work?
+
+## You have examples or an idea how it can be implemented?
```

Comment:
Making it a bit more friendly: Do you have any examples of how you would like to see us implement it?